google/cloud/spanner_v1/transaction.py

```diff
@@ -268,13 +277,18 @@ def batch_update(self, statements):
             self._execute_sql_count + 1,
         )
 
-        response = api.execute_batch_dml(
-            session=self._session.name,
-            transaction=transaction,
-            statements=parsed,
-            seqno=seqno,
-            metadata=metadata,
-        )
+        trace_attributes = {
+            # Get just the queries from the DML statement batch
+            "db.statement": [statement["sql"] for statement in parsed]
```

Comment:
Previously, `sql` in `trace_attributes = {"db.statement": sql}` is a string. Here, we pass a list of strings as a parameter. Will this cause an issue?

Reply:
Good point, not sure what the right answer is - could remove it, only publish the first statement, or merge them together?

Reply:
Probably merge them together and use `;` for a delimiter?

Reply:
> Probably merge them together and use ; for a delimiter?

This option makes the most sense :+1:

Reply:
done